### ISSUE TITLE
Accessibility - Dataverse - Color Changes

### ISSUE THREAD

**Issue by adam3smith** [QualitativeDataRepository/dataverse#75]
The following color switches should happen across the CSS:

**Already on Dev**
#4B74FF -->  #0039F5
#4B75FF  --> #0039F5
#767676 --> #424242
#969696 --> #666666

**in styles.css** 
#777777 --> #424242 
#777 --> #424242 (also in text.emhasis.less -- not sure whether that's easier to change or override?)

#979797 --> #666666

structure.css
#808080 --> #666666

**Comment by qqmyers** [comment]
I think the .less 'files' are all part of the bootstrap.* files in the Dataverse source, so I think that changing all the `#777` occurrences would work. That said, I think we pick these up via things like the label-default class and

-  we could change the class we use
- we could override the label-default color in our own css classes
- we could assume that bootstrap is handling contrast in things like labels and not change it (though these colors wouldn't then be matched elsewhere in the UI

Otherwise, I should be able to make the other changes and get them out to dev/stage this weekend.

**Comment by adam3smith** [comment]
The third  is not an option -- the `#777` shows up e.g. as the date of all projects in the browse view.
![image](https://user-images.githubusercontent.com/128986/133838338-ab149536-0961-479e-8bbe-8fe48c050b68.png)

I think overriding this in our own CSS would be best

**Comment by qqmyers** [comment]
Fixed this one with 
.text-muted {
  color: #424242;
}

Also making other color changes on dev (search text/placeholder, Dataverse affiliation (BoB U on dev/main), warning labels to address Axe warnings. It looks like the contrast requirement is text-size sensitive so if you want other colors/want to increase size, etc. instead, let me know.

**Comment by qqmyers** [comment]
Note that the Affiliation (and Tagline) color can come from the Theme/text color, which overrides the default in the code, so that ~should be updated when we move to other machines (I don't think we have Affiliation/Tagline info aside from dev though.)

**Comment by adam3smith** [comment]
yup -- also need to check that on the sub collections

**Comment by qqmyers** [comment]
found/changed the ETS tag color on prod - I think that's the only one left that had a color issue.
